### PR TITLE
ETCD-187: add dashboards CPU iotwait on master nodes

### PR DESCRIPTION
**Pull request by Elbehery**
resolves https://issues.redhat.com/browse/ETCD-187

### CODE REVIEW

File: jsonnet/cpu_iowait_dashboard.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+          "legendFormat": "{{instance}} cpu iowait",
+          "refId": "A",
+          "step": "4"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "Panel Title",
```
Comment (tjungblu) [inline jsonnet/cpu_iowait_dashboard.json:73]: maybe a more descriptive title? how about the same as the dashboard title?

**Review by tjungblu — CHANGES_REQUESTED** [review]
mind if you create a screenshot? just created another clusterbot for a bugfix

**Comment by Elbehery** [comment]
> mind if you create a screenshot? just created another clusterbot for a bugfix

yes i tried but it always fails, creating a new one now :)

File: jsonnet/cpu_iowait_dashboard.json (hunk 1 above)
Comment (Elbehery) [reply jsonnet/cpu_iowait_dashboard.json]: done

**Comment by Elbehery** [comment]
Screenshot 2023-09-25 at 14 31 31
Screenshot 2023-09-25 at 14 31 55

File: manifests/0000_90_cluster-etcd-operator_01-dashboards.yaml [hunk 2]
```diff
@@ -1438,6 +1438,158 @@ data:
...
+         ],
+         "thresholds": [
+
+         ],
+         "timeFrom": null,
+         "timeRegions": [
+
+         ],
+         "timeShift": null,
+         "title": "Panel Title",
```
Comment (tjungblu) [inline manifests/0000_90_cluster-etcd-operator_01-dashboards.yaml:1526]: still a panel title left

Comment (tjungblu) [reply manifests/0000_90_cluster-etcd-operator_01-dashboards.yaml:1526]: guess you need to make update?

Comment (Elbehery) [reply manifests/0000_90_cluster-etcd-operator_01-dashboards.yaml:1526]: yes :D

Comment (Elbehery) [reply manifests/0000_90_cluster-etcd-operator_01-dashboards.yaml]: done

**Comment by tjungblu** [comment]
/lgtm

well done @Elbehery

**Pull request merged**